openslides/utils/autoupdate.py

```diff
@@ -92,6 +94,12 @@ def ws_add_site(message):
         access_permissions = collection.get_access_permissions()
         restricted_data = access_permissions.get_restricted_data(collection, user)
 
+        if restricted_data is None:
+            continue
+
+        # At this point restricted_data has to be a list. So we have to tell it mypy
+        restricted_data = cast(List[Dict[str, Any]], restricted_data)
```

Comment:
Please check if this is still required.

Reply:
Yes it is. mypy can not know, that at this point restricted_data is a list. ```cast()``` is there for situations like this.